src/chef-mover/src/mover_global_admins_user_addition_callback.erl

```diff
@@ -0,0 +1,106 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92 -*-
+%% ex: ts=4 sw=4 et
+%% @author Steven Danna <steve@chef.io>
+%% @copyright 2015 Chef Software, Inc.
+%%
+%% The global_admins_user_addition addition callback adds the user
+%% group from a given organization into the ORGNAME_global_admins
+%% group for that organization.
+%%
+%% The ORGNAME_global_admins group has a signle purpose: providing
+%% READ access to other user objects in ORGNAME.  Thus, the only
+%% change that this migration produces is that users that share an org
+%% can now see each others user objects.
+%%
+%% This change is necessary to facilitate the use of chef-vault.
+%%
+-module(mover_global_admins_user_addition_callback).
+
+-export([
+         migration_init/0,
+         migration_complete/0,
+         migration_type/0,
+         supervisor/0,
+         migration_start_worker_args/2,
+         migration_action/2,
+         next_object/0,
+         error_halts_migration/0,
+         reconfigure_object/2,
+         needs_account_dets/0
+        ]).
+
+-include("mover.hrl").
+
+-record(org, {name, id, authz_id}).
+-record(group, {name, id, authz_id}).
+
+migration_init() ->
+    mv_oc_chef_authz_http:create_pool(),
+    mover_transient_migration_queue:initialize_queue(?MODULE, all_orgs()).
```

Comment:
Thinking of our hosted run, we might need to give some thought to how we initialize this. If we have to pause and rerun, we'll be doing a lot of duplicated attempts here. (On the other hand, that may be OK.)